releases/2021.02.yaml

```diff
@@ -183,3 +183,4 @@ repositories:
     type: git
     url: https://github.com/robotology/diagnostic-daemon.git
     version: v1.0.0
+
```

Comment:
`osqp-matlab` disappeared after the merge from master. Was this intended?

Reply:
Yes. The PR was opened before the 2021.02 release, but as the 2021.02 release has passed, it does not make sense to add a release for osqp-matlab here. Furthermore, we now have the latest releases file: https://github.com/robotology/robotology-superbuild/pull/596 , so once osqp-matlab has a release we can add it there.

Reply:
Indeed, I had seen that one. Great.

Reply:
> once osqp-matlab has a release we can add it there.

Do we do it in a new issue or we hold this PR until then?

Reply:
I think we can review https://github.com/dic-iit/osqp-matlab-cmake-buildsystem/pull/2 and then I will release a first version of osqp-matlab so that we can add its release here.

Reply:
Ok. I took a look at `robotologyUpdateLatestReleases.sh` and saw that in this line
https://github.com/robotology/robotology-superbuild/blob/ac6952e5f51d000a7e609e4d9684aae123db3752/scripts/robotologyUpdateLatestReleases.sh#L72
it assumes that the entry for the package exists, so if I understood it well, I guess we'll have to add it manually the first time.

Reply:
looking at https://github.com/dic-iit/osqp-matlab-cmake-buildsystem/pull/2 now...

Reply:
>  it assumes that the entry for the package exists, so if I understood it well, I guess we'll have to add it manually the first time.

Yes!